content/_changelogs/12.1.0.md

```diff
@@ -0,0 +1,35 @@
+## 12.1.0
+
+_Released 12/12/2022_
+
+**Features:**
+
+- Added [`cy.getAllCookies()`](/api/commands/getallcookies) and
+  [`cy.clearAllCookies()`](/api/commands/clearallcookies), which get and clear
+  cookies for all browser cookies across any domains. Addresses
+  [#24265](https://github.com/cypress-io/cypress/issues/24265),
+  [#8956](https://github.com/cypress-io/cypress/issues/8956), and
+  [#408](https://github.com/cypress-io/cypress/issues/408)
+- Updated the error messaging to provide more context when parallel group
+  parameters are mismatched when sent to the Cypress Cloud. Fixes
+  [#24734](https://github.com/cypress-io/cypress/issues/24734)
+
+**Bugfixes:**
+
+- Fixed an issue where recent versions of Firefox in headless mode would behave
+  inconsistently with headed. Fixes
+  [#24900](https://github.com/cypress-io/cypress/issues/24900)
+- Fixed an issue where `.select(index)` would fail when multiple `<option>`
```

Comment:
like to select command

Reply:
Updated: [`ff847db` (#4920)](https://github.com/cypress-io/cypress-documentation/pull/4920/commits/ff847db3e9d7c80fe99c140c978f5a2a2d7d822c)